Source/ACE.Entity/Enum/EquipmentSet.cs

```diff
@@ -16,22 +18,39 @@ public enum EquipmentSet
         ArmMindHeart                     = 10,
         ArmorPerfectLight                = 11,
         ArmorPerfectLight2               = 12,
+        [Description("Soldier's")]
```

Comment:
should not be using reflection lookups here
reflection lookups are slow, and this pattern should be avoided

Reply:
Might have to see if there is a better way to insert the apostrophe into the name of the first six enum names

Reply:
removed; solved with string manipulation